Index: templates/Starter.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/Starter.py b/templates/Starter.py
new file mode 100644
--- /dev/null	(date 1632167696857)
+++ b/templates/Starter.py	(date 1632167696857)
@@ -0,0 +1,18 @@
+from flask import Blueprint, render_template
+from algorithm.image import image_data
+
+starter_bp = Blueprint('starter', __name__,
+                       url_prefix='/starter',
+                       template_folder='templates',
+                       static_folder='static',
+                       static_url_path='assets')
+
+
+@starter_bp.route('/binary/')
+def binary():
+    return render_template("starter/binary.html")
+
+
+@starter_bp.route('/rgb/')
+def rgb():
+    return render_template('starter/rgb.html', images=image_data())
